packages/amplify-graphql-index-transformer/src/__tests__/__snapshots__/amplify-graphql-index-transformer.test.ts.snap

```diff
@@ -3531,139 +3531,6 @@ $util.toJson({})
 #set( $filterArgsMap = {} )
 ## [End] Set map initialization for @key **
 ## [Start] Set query expression for @key **
-$util.qr($QueryMap.put('status+createdAt' , 'ByStatus'))
```

Comment:
I'm not really able to tell from this test on it's own, but to clarify, what's being kept here is the customer overridden logic only, not the one we'd write?

Reply:
That's right. The customer overridden logic will replace the one we create.